examples/guestbook-operator/controllers/guestbook_controller_test.go

```diff
@@ -27,6 +27,8 @@ import (
 func TestGuestbook(t *testing.T) {
 	v := golden.NewValidator(t, api.SchemeBuilder)
 	dr := &GuestbookReconciler{
+		//Client: v.Client(),
+		// TODO: Uncomment line above, remove the line below to use latest version of kubebuilder-declarative-pattern
 		Client: v.Manager().GetClient(),
```

Comment:
How to run these tests with the current version of library? If I add `replace sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20210922163802-cac4a6cf1977 => ../../` to `go.mod`, [the tests fail](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kubebuilder-declarative-pattern/203/pull-declarative-test/1488518439340347392) because:
> go: updates to go.mod needed; to update it:
>	go mod tidy

Running the tests locally with these changes succeeds.